content/docs/apps/managing-teammates.md

```diff
@@ -19,6 +19,22 @@ If you invited them with an agency email address and they're part of an agency t
 
 After a teammate gets a cloud.gov account, an Org Manager or Space Manager will need to give them roles in your orgs and spaces so they can collaborate with you. For details about how org and space roles and permissions work, see [Cloud Foundry roles and permissions](http://docs.cloudfoundry.org/concepts/roles.html#roles).
 
+Org Managers can assign roles using the dashboard or command line. Space Managers can assign roles using the command line.
+
+{{% dashboard %}}
+
+If you're an Org Manager, go to your dashboard and click on the name of your org.
+
+![User roles](/img/user-role-management.png)
+
+Go to the organization where you have Org Manager permissions.
+
+On the Organization page, scroll down to the "Organization users" section. From there, you will notice a list of all the organization users and their organization roles. The user roles can be changed by clicking the checkboxes beside their names.
```

Comment:
Instead of telling people what they'll see ("From there, you will notice"), you can more simply say "This provides". Also, instead of passive voice ("The user roles can be changed"), you can use active voice and speak directly to the reader: "You can change the user roles".

Also, if they're the only person in the org, they won't see a list of users - they'll just see "You are the only user in this organization". Not sure how you want to handle that case in the text here.